tools/ci_connector_ops/tests/test_qa_checks.py

```diff
@@ -6,87 +6,75 @@
 from pathlib import Path
 
 import pytest
-
 from ci_connector_ops import qa_checks, utils
 
 
-@pytest.mark.parametrize("connector, expect_exists", [
-    (utils.Connector("source-faker"), True),
-    (utils.Connector("source-foobar"), False),
-])
+@pytest.mark.parametrize(
+    "connector, expect_exists",
+    [
+        (utils.Connector("source-faker"), True),
+        (utils.Connector("source-foobar"), False),
+    ],
+)
```

Comment:
Lots of changes from `black` - my precommit was on vacation in the last couple of weeks :( But it's back!